src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,33 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+    if(typeof text==="string"){
```

Comment:
Wrapping your whole program in an `if...else` makes it harder to follow especially when the program is long. Learn about `Early Return`.